plugin/plugin.go

```diff
@@ -125,13 +144,49 @@ func Exec(ctx context.Context, args Args) error {
 					URL:         deeplink,
 				},
 				Environment: Environment{
-					ID:          environ,
+					ID:          environmentID,
 					Displayname: environ,
-					Type:        environ,
+					Type:        environmentType,
 				},
 			},
 		},
 	}
+	if len(args.IssueKeys) > 0 {
+		deploymentPayload.Deployments[0].Associations = nil
+	}
+	/*fmt.Println("formatted deploymentPayload JSON data")
```

Comment:
remove this comment please. Please make sure those debug/temp/commented out code doesn't appear in final PR

Reply:
done